### PR TITLE
Dit 715 ui changes

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

**Comment by SteveSugden** [comment]
This is the PR to fix the acceptance tests -0 https://github.com/hmrc/digital-tariffs-acceptance-tests/pull/106

File: app/views/components/input_textarea.scala.html [hunk 1]
```diff
@@ -29,6 +30,9 @@
         @hint.map { hint =>
             <span class="form-hint">@hint</span>
         }
+        @htmlHint.map { htmlHint =>
+            <span class="form-hint">@htmlHint</span>
```
Comment (ed0906) [inline app/views/components/input_textarea.scala.html]: Minor, but is a span the right html element here? Looks more like a `div` scenario to me

File: app/views/confirmation.scala.html [hunk 2]
```diff
@@ -23,34 +22,39 @@
...
     bodyClasses = None) {
 
-	  <div class="govuk-box-highlight">
-			@components.heading("confirmation.heading", "heading-xlarge")
-      <p class="font-large">
-    		@messages("confirmation.yourRefNumber")<br>
-        <span id="confirmation-reference" class="bold">@messages("confirmation.refValue", confirmation.reference)</span>
-      </p>
-	  </div>
+    <div class="govuk-box-highlight">
```
Comment (ed0906) [inline app/views/confirmation.scala.html]: Does this class do anything?
Generally we don't have access to `govuk-` classes from the design system

File: app/views/components/input_textarea.scala.html (hunk 1 above)
Comment (SteveSugden) [reply app/views/components/input_textarea.scala.html]: You have a good point.  I'll change

**Pull request merged**